app/i18n/locales/en-US.json

```diff
@@ -117,7 +117,7 @@
   "wallet.restore.dialog.wallet.name.input.label": "Wallet name",
   "wallet.send.form.amount.equalsAda": "equals {amount} ADA",
   "wallet.send.form.amount.hint": "Amount in Lovelaces",
-  "wallet.send.form.amount.label": "Amount in Lovelaces",
+  "wallet.send.form.amount.label": "Amount",
   "wallet.send.form.description.hint": "You can type a message here, if you want.",
```

Comment:
@darko-mijic as agreed during our discussion today, this is not needed as Japanese translation actually contains translation for "Amount" word and not "Amount in Lovelaces": https://github.com/input-output-hk/daedalus/blob/cardano-sl-0.4-rc/app/i18n/locales/ja-JP.json#L120